pkg/plugins/plugins.go

```diff
@@ -106,6 +118,86 @@ func (p PluginDTO) IncludedInSignature(file string) bool {
 	return true
 }
 
+func (p PluginDTO) Markdown(name string) []byte {
+	// nolint:gosec
+	// We can ignore the gosec G304 warning since we have cleaned the requested file path and subsequently
+	// use this with a prefix of the plugin's directory, which is set during plugin loading
+	path := filepath.Join(p.pluginDir, mdFilepath(strings.ToUpper(name)))
+	exists, err := fs.Exists(path)
+	if err != nil {
+		return make([]byte, 0)
+	}
+	if !exists {
+		path = filepath.Join(p.pluginDir, mdFilepath(strings.ToLower(name)))
+	}
+
+	exists, err = fs.Exists(path)
+	if err != nil || !exists {
+		return make([]byte, 0)
+	}
+
+	// nolint:gosec
+	// We can ignore the gosec G304 warning since we have cleaned the requested file path and subsequently
+	// use this with a prefix of the plugin's directory, which is set during plugin loading
+	data, err := os.ReadFile(path)
+	if err != nil {
+		return make([]byte, 0)
+	}
+	return data
+}
```

Comment:
Possible to get rid of the nolint using what we've used for plugin dashboards?
https://github.com/grafana/grafana/blob/13d5ad2ce2a7283dc25dace1328cbf0c11fec639/pkg/plugins/manager/dashboards/filestore.go#L87-L94